spacecmd/src/spacecmd/utils.py

```diff
@@ -682,9 +683,20 @@ def parse_api_args(args, sep=','):
     except ValueError:
         ret = [parse_str(a) for a in parse_list_str(args, sep)]
 
+    ret = datetime_parser_lst(ret)
     return ret
 
 
+def datetime_parser_lst(lst):
+    for i, l in enumerate(lst):
+        if isinstance(l, str):
+            try:
+                lst[i] = parse_datetime(l)
+            except ParserError:
+                pass
+    return lst
```

Comment:
small question about the design: is it intended that this function both modifies the passed list and return it? Both of these ways of calling are basically the same:
```python
old = ["2022-06-08"]
datetime_parser_lst(old)
assert old[0] == datetime.datetime(2022, 6, 8, 0, 0)

also_old = ["2022-06-08"]
new = datetime_parser_lst(also_old)
assert new[0] == datetime.datetime(2022, 6, 8, 0, 0)

Reply:
You're right. Maybe it's better not returning it and just mutate the parameter in this case.